src/racer/matchers.rs

```diff
@@ -298,7 +300,7 @@ pub fn match_mod(msrc: Src, blobstart: usize, blobend: usize,
                 contextstr: filepath.to_str().unwrap().to_owned(),
                 generic_args: Vec::new(),
                 generic_types: Vec::new(),
-                docs: String::new(),  //TODO: fix
+                docs: String::new(),
```

Comment:
Isn't an empty string arguably the same as a None? There's never going to be a zero length doc string, so assuming that an empty string means no documentation sounds like a fair trade off between correctness and ease of use of the interface. Would still be happy to hear a different argument :)